app/src/main/AndroidManifest.xml

```diff
@@ -150,6 +150,10 @@
                         android:host="conversation"
                         android:scheme="wire" />
 
+                <data
+                        android:host="user"
+                        android:scheme="wire" />
+
```

Comment:
It is for user profile :)  In fact, we can not change, since this is the deep link coming from web implemented in ios and desktop, for example. 

https://wire-account-staging.zinfra.io/user-profile/?id=61207d7e-6dd2-4e51-9128-6f6a3ce51ec0